### ISSUE TITLE
Error parsing LocalForward statement

### ISSUE THREAD

**Issue by wilriker** [ObserverOfTime/tree-sitter-ssh-config#3]
Similar to #1 the parser fails to handle `LocalForward` statements like

```sshconfig
Host host
	LocalForward /home/me/some.sock /var/run/socks/sock.sock
	LocalForward 8080 localhost:80
	LocalForward 8443 localhost:443
```

**Comment by wilriker** [comment]
Sorry for the noise. This one was probably also already fixed with #1 just `nvim-treesitter` did not yet have the newest version.

**Comment by ObserverOfTime** [comment]
nvim-treesitter will be updated soon.
